src/portal/azext_portal/azext_metadata.json

```
@@ -0,0 +1,4 @@
+{
+    "azext.isPreview": true,
+    "azext.minCliCoreVersion": "2.0.67"
```

Requested by PM, we release all Breadth Coverage extensions initially with an Experimental tag and use 2.3.1 as the minimal version which added the support for this tag. 
```suggestion
    "azext.isExperimental": true,
    "azext.minCliCoreVersion": "2.3.1"
```